src/it/scala/net/snowflake/spark/snowflake/SnowflakeResultSetRDDSuite.scala

```diff
@@ -1439,7 +1439,7 @@ class SnowflakeResultSetRDDSuite extends IntegrationSuiteBase {
     }
   }
 
-  test("test upload with AWS multiple parts upload API") {
+  ignore("test upload with AWS multiple parts upload API") {
```

Comment:
could you add comment here, to describe why we ignore this test and do we have plan to re-enable it?